stdlib/websub/src/test/java/org.ballerinalang.net.websub/WebSubHubStartUpTest.java

```diff
@@ -87,7 +87,8 @@ public void testHubShutdownAndStart() {
 
     @AfterClass
     public void tearDown() {
-        BRunUtil.invoke(result, "stopHub", new BValue[]{hubStartUpObject});
+        //TODO Uncomment following once test suite is migrated to jBal types and values
+//        BRunUtil.invoke(result, "stopHub", new BValue[]{hubStartUpObject});
```

Comment:
this needs to change?

Reply:
yes, until we get proper jvm value test suite, we can pass a return value again to another class as it goes through few conversions